backend/fms_core/viewsets/full_sample.py

```diff
@@ -0,0 +1,216 @@
+    serializer_class = FullSampleSerializer
+
+    ordering_fields = (
+        *_list_keys(_full_sample_filterset_fields),
+    )
+
+    filterset_fields = {
+        **_full_sample_filterset_fields,
+    }
+
+    template_action_list = [
+        {
+            "name": "Add Samples",
+            "description": "Upload the provided template with up to 384 new samples.",
+            "template": SAMPLE_SUBMISSION_TEMPLATE,
+            "importer": SampleSubmissionImporter,
+        },
+        {
+            "name": "Update Samples",
+            "description": "Upload the provided template with up to 384 samples to update.",
+            "template": SAMPLE_UPDATE_TEMPLATE,
+            "importer": SampleUpdateImporter,
+        }
+    ]
+
+    def create(self, request, *args, **kwargs):
+        error = {}
+        full_sample = request.data
+
+        #Container related information
+        try:
+            container_obj = Container.objects.get(pk=full_sample['container'])
+        except Exception as err:
+            error['container'] = err
+
+        #Individual related information
+        individual_obj = None
+        if full_sample['individual']:
+            try:
+                individual_obj = Individual.objects.get(pk=full_sample['individual'])
+            except Exception as err:
+                error['individual'] = err
+
+        #Sample Kind related information
+        try:
+            sample_kind_obj = SampleKind.objects.get(pk=full_sample['sample_kind'])
+        except Exception as err:
+            error['sample_kind'] = err
+
+        biosample_data = dict(
+            collection_site=full_sample['collection_site'],
+            **(dict(individual=individual_obj) if individual_obj is not None else dict()),
+            **(dict(alias=full_sample['alias']) if full_sample['alias'] is not None else dict()),
+        )
+
+        try:
+            biosample = Biosample.objects.create(**biosample_data)
+
+            derived_sample_data = dict(
+                biosample_id=biosample.id,
+                sample_kind=sample_kind_obj,
+                **(dict(tissue_source=full_sample['tissue_source']) if full_sample['tissue_source'] is not None else dict()),
+            )
+            if full_sample['experimental_group']:
+                derived_sample_data['experimental_group'] = json.dumps([
+                    g.strip()
+                    for g in RE_SEPARATOR.split(full_sample['experimental_group'])
+                    if g.strip()
+                ])
+
+            derived_sample = DerivedSample.objects.create(**derived_sample_data)
+
+            sample_data = dict(
+                name=full_sample['name'],
+                volume=full_sample['volume'],
+                creation_date=full_sample['creation_date'],
+                container=container_obj,
+                comment=(full_sample['comment'] or (f"Automatically generated on {datetime.utcnow().isoformat()}Z")),
```

Comment:
why change the input of the user ? All field should be exactly as submitted (other than the possible small formatting)... It was not there before i do not see why it need to be there now ... Users expect to have an empty comment inserted ...